Jellyfin.Api/Jellyfin.Api.csproj

```diff
@@ -7,10 +7,15 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
Why are you only doing it for the API project?

Reply:
It's turned on for every project, this one when it was created did not have this properly done, breaking the versioning. (it was 1.0.0.0)